helpers/deploy/deploymentUtils.ts

```diff
@@ -14,20 +15,48 @@ export const getAcmAdminAccount = async (network: SUPPORTED_NETWORKS): Promise<s
     return "0x285960C5B22fD66A736C7136967A3eB15e93CC67"; // ETHEREUM MULTISIG
   } else if (network === "opbnbtestnet") {
     return "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf"; // OPBNBTESTNET MULTISIG
+  } else if (network === "opbnbmainnet") {
+    return "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207"; // OPBNBMAINNET MULTISIG
   }
-
-  const normalTimelock = await ethers.getContract("NormalTimeLock");
+  const normalTimelock = await ethers.getContract("NormalTimelock");
   return normalTimelock.address;
 };
 
+export const getOmnichainGovernanceExecutorAdminAccount = async (network: SUPPORTED_NETWORKS): Promise<string> => {
```

Comment:
It seems like this and the ACM admin could be combined. The addresses are the same

Reply:
After the Timelocks are finalized on the remote chain, we will set the admin of the ACM to Timelock.